### PR TITLE
update integration tests

### PR DESCRIPTION
**Pull request by Nesurion**
Adds integration tests for the update command.

### CODE REVIEW

**Comment by Nesurion** [comment]
@giantswarm/cena RFR

File: int-tests/002-update.t [hunk 1]
```diff
@@ -0,0 +1,47 @@
+Create a group to play around with.
+  $ mkdir update-group
+  $ touch update-group/update-group-foo@.service
```
Comment (JosephSalisbury) [inline int-tests/002-update.t]: Do you need the touches?

**Comment by JosephSalisbury** [comment]
IIWFYLGTM

File: int-tests/002-update.t (hunk 1 above)
Comment (zeisss) [inline int-tests/002-update.t]: he shouldn't need them, afaict

**Pull request merged**